### PR TITLE
[#173702628] Reworks accordion component and improves accessibility

### PR DESCRIPTION
**Pull request by CrisTofani**
**Short description:**
This PR adds a new Accordion component with improved accessibility for FAQ component

### CODE REVIEW

File: ts/components/ui/Accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            ]}
+          />
+        </View>
+        {!expanded && <ItemSeparatorComponent noPadded={true} />}
+      </TouchableDefaultOpacity>
+    );
+  };
+
+  const renderContent = (content: string) => (
+    <View style={styles.pad}>
```
Comment (fabriziofff) [inline ts/components/ui/Accordion.tsx:77]:
```suggestion
    <View style={styles.pad} accessible={expanded}>
```
On ios, if I open and close the accordion and after that I use the right swipe gesture, the focus is set on the content of the closed accordion.

Comment (CrisTofani) [reply ts/components/ui/Accordion.tsx:77]: I can't test it on iOS but what do you think of `<View style={styles.pad} accessible={true} accessibilityElementsHidden={expanded}>`
?

Comment (CrisTofani) [reply ts/components/ui/Accordion.tsx]: the fix is in commit 5f341b7

File: locales/it/index.yml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ---
 global:
   accessibility:
+    expanded: Espanso
```
Comment (Undermaken) [inline locales/it/index.yml:4]:
```suggestion
    expanded: Paragrafo espanso
```
Could it be clearer we have a content that could be closed/opened?
Perhaps `Sezione` could be better?

Comment (CrisTofani) [reply locales/it/index.yml:4]: Maybe Paragraph better explains it has an expanded text below, I'll apply this changes in the next commit

Comment (CrisTofani) [reply locales/it/index.yml]: @Undermaken updated in 3f1674b

**Pull request merged**